.circleci/config.yml

```diff
@@ -110,26 +110,6 @@ commands:
 #
 
 jobs:
-  docs-python36:
-    docker:
-      - image: circleci/python:3.6
-    steps:
-      - checkout
-
-      - apt-install
-      - fonts-install
-      - pip-install
-      - deps-install:
-          numpy_version: "==1.13.0"
```

Comment:
Should leave a minimum NumPy in 3.7 build?

Reply:
We have a test with the minimum numpy and friends on travis.  Are we worried about the docs having issues when the tests pass on a given numpy version?

Reply:
Sometimes examples do weird things, so not sure. We could just say docs must be built on newer versions?

Reply:
well, if the docs _do_ find weird behavior with old versions of numpy that is identifying a gap in our testing and is something we should fix 2x...

Reply:
I put back a py38 + numpy1.16 build.

Reply:
Circle maybe doesn't like that.